test/e2e-api/nats.go

```diff
@@ -149,5 +172,6 @@ func (n *NatsController) PumpJetstream(w http.ResponseWriter, r *http.Request) {
 }
 
 func (n *NatsController) Close() {
+	// no lazy clozing required becuase they check if nc is not null before they close
```

Comment:
Typos
// no lazy closing is required here because the client.Close()  functions does a nil check before closing.